### DISCUSSION TITLE
ByeByeDPI v1.3.8 Beta3

### DISCUSSION THREAD

**Discussion by oren74** [dovecoteescapee/ByeDPIAndroid#192] [Q&A]
у меня не ставиться на ми stik4k ByeByeDPI v1.3.8 Beta3 ,как поставить????

**Comment by Biletvray** [comment]
Где его скачать то ссылок нет

**Reply by Trigger239** [reply]
[Здесь](https://github.com/romanvht/ByeDPIAndroid/tags)

**Comment by AlcherS1** [comment]
Приставка Ugoos. Все равно какой провайдер.  Потом ByeByeDPI 1.3.8. Перебор параметров в ком строке, см.. на процент успеха.  Сначала открываете youtube из google play. Youtube tv ( 4pda youtube tv) работает на приставке если сначала открыть youtube из google play. Потом youtube tv чтобы удобно открывать пультом.

**Comment by BloodRomance** [comment]
Спасибо, уважаемый разработчик за чудесную программу!
Есть вопрос, возможно сможете помочь - в инете инфы 0...
Как заставить работать программу с Orbot, а точнее подскажите, если не трудно, как настроить Orbot с данной прогой?
Иногда нужен ВПН, предпочитаю именно орбот, т.к. нет заморочек с выбором сервера и тд.
Но как заставить работать его в режиме прокси так и не понял...
Заранее благодарен

**Comment by olvol** [comment]
В чем может быть проблема? Программа установлена на нвидия шилд ТВ про. Через несколько секунд после подключения происходит автоматический сбросится соединения. Приставку перезагружал, прогу переустанавливал. Ничего не помогает

**Comment by Cagors** [comment]
Неплохо было бы реализовать возможность динамически менять профили  настроек и/или автоматическое отключение/включение по заданному или рандомному таймеру, ибо складывается впечатление, что dpi научились  определять и, если он включен постоянно, то срабатывает блокировка и скорость падает почти до нуля. А если вручную включать/выключать, с некоторыми промежутками по времени, то всё грузится стабильно, но это, по понятным причинам, очень неудобно.

**Comment by AlcherS1** [comment]
У нас хорошо работает уже давно последняя версия на приставке Ugoos android TV, роутер Keenetic. Один раз настроили через подбор командной строки с высоким процентом и забыли на несколько месяцев.
